packages/relay/test/synchronizer.test.ts

```diff
@@ -10,6 +10,109 @@ import { signUp } from './utils/signUp'
 import { HTTP_SERVER } from './configs'
 import { io } from 'socket.io-client'
 import { post } from './utils/post'
+import IpfsHelper from '../src/services/utils/IpfsHelper'
+
+describe('Synchronize Post Test', function () {
+    let snapshot: any
+    let express: ChaiHttp.Agent
+    let sync: UnirepSocialSynchronizer
+    let unirep: Unirep
+    let unirepApp: UnirepApp
+    let users: {
+        hashUserId: String
+        wallet: any
+        userState: UserState
+    }[] = []
+
+    before(async function () {
```

Comment:
Seems like most of the test setup is the same between Post and Comment. We can consider to separate it into another utils file for this.

Reply:
I will leave this currently and merge first.